docs/public_code.md

```diff
@@ -0,0 +1,188 @@
+Any source code not currently in use (such as new versions, proposals or older versions) SHOULD be published. | yes | [releases](https://github.com/open-zaak/open-zaak/releases), [docker hub tags](https://hub.docker.com/r/openzaak/open-zaak/tags)
+The source code MAY provide the general public with insight into which source code or policy underpins any specific interaction they have with your organization. | |
+
+
+## [Bundle policy and source code](https://standard.publiccode.net/criteria/bundle-policy-and-code.html)
+
+- [x] Policy bundled, tested, and linked in the documentation, some policy not available in English
+
+Requirement | meets | links and notes
+-----|-----|-----
+A codebase MUST include the policy that the source code is based on. | yes | [API specs embeded in the codebase](https://github.com/open-zaak/open-zaak/blob/master/src/openzaak/components/zaken/openapi.yaml), and linked from [documentation](https://github.com/open-zaak/open-zaak)
+A codebase MUST include all source code that the policy is based on. | | "yes" or "not applicable", do we consider it based on the API?
+All policy and source code that the codebase is based on MUST be documented, reusable and portable. | yes | code dependencies are OSI
+Policy SHOULD be provided in machine readable and unambiguous formats. | yes | OpenAPI is in machine readable yaml format
+Continuous integration tests SHOULD validate that the source code and the policy are executed coherently. | yes | [Travis CI config](https://github.com/open-zaak/open-zaak/blob/master/.travis.yml#L86), [API Test Platform](https://api-test.nl/server/1/224fd5be-bc64-4d55-a190-454bee3cc8e3/14bc91f7-7d8b-4bba-a020-a6c316655e65/latest/)
+
+## [Create reusable and portable code](https://standard.publiccode.net/criteria/reusable-and-portable-codebases.html)
+
+- [x] designed to be reusable from the start
+
+Requirement | meets | links and notes
+-----|-----|-----
+The codebase MUST be developed to be reusable in different contexts. | yes | Designed to be so from the start.
+The codebase MUST be independent from any secret, undisclosed, proprietary or non-open licensed code or services for execution and understanding. | yes | installation supports docker, kubernetes, vmware appliances, bare-metal is possible
+The codebase MUST be in use by multiple parties. | yes | Deployed in multiple sandbox environments (e.g: Utrecht is testing it, others Den Haag, Delft looking at it.)
+The roadmap SHOULD be influenced by the needs of multiple parties. | yes | [Dimpact](https://www.dimpact.nl/openzaak), [market consultation](https://github.com/open-zaak/open-zaak-market-consultation) | Code SHOULD be general purpose and SHOULD be configurable. |
```

Comment:
There is a missing line break here, making one requirement hidden.

Reply:
fixed